configure.py

```diff
@@ -155,10 +155,10 @@ def _gen_android_cross_file(cfg):
         sha256="8e6efa8d9ec5ef1b2c735b2fc2afdfed210a3a5aa01c48e572326914e27bb221",
     ),
     sdl2_Windows=dict(
-        version="2.26.5",
+        version="2.30.1",
```

Comment:
Sorry for the late review; 2.30.2 was released 2 days ago

Reply:
Updated, thanks !